src/SDKs/ContainerInstance/Management.ContainerInstance/Generated/Models/OperationDisplay.cs

```diff
@@ -50,28 +50,28 @@ public OperationDisplay(string provider = default(string), string resource = def
         partial void CustomInit();
 
         /// <summary>
-        /// Gets or sets the name of the provider of the operation.
+        /// Gets the name of the provider of the operation.
         /// </summary>
         [JsonProperty(PropertyName = "provider")]
-        public string Provider { get; set; }
+        public string Provider { get; private set; }
 
         /// <summary>
-        /// Gets or sets the name of the resource type of the operation.
+        /// Gets the name of the resource type of the operation.
         /// </summary>
         [JsonProperty(PropertyName = "resource")]
-        public string Resource { get; set; }
+        public string Resource { get; private set; }
 
         /// <summary>
-        /// Gets or sets the friendly name of the operation.
+        /// Gets the friendly name of the operation.
         /// </summary>
         [JsonProperty(PropertyName = "operation")]
-        public string Operation { get; set; }
+        public string Operation { get; private set; }
```

Comment:
This could potentially be a breaking change for customers. Please bump the package version number to `1.1.0`